PSPs/drafts/psp-17.md

```diff
@@ -56,13 +54,18 @@ pub trait IPSP20 {
  fn decrease_allowance(&mut self, spender: AccountId, delta_value: Balance);
 }
 
-pub trait IPSP20Metadata {
+pub trait IPSP17Metadata {
  fn token_name(&self) -> Option<String>;
 
  fn token_symbol(&self) -> Option<String>;
 
  fn token_decimals(&self) -> u8;
 }
+
+pub trait IPSP17Receiver {
+ fn on_psp17_received(&mut self, operator: AccountId, from: AccountId, value: Balance, data: Vec<u8>) -> Result<(), PSP17ReceiverError>;
```

Comment:
You need to describe behavior of this trait somewhere. For example like in solidity via comments